src/style.css

```diff
@@ -1 +1,107 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+  min-width: 1024px;
+}
+
+.header {
+  position: sticky;
+  top: 0;
+  --height: 60px;
+  width: 100%;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.blockHeader {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100%;
+  padding-right: 50px;
+  padding-left: 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.logo:hover {
+  outline: 2px solid #00acdc;
+}
+
+.nav {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item {
+  line-height: var(--height);
+  height: var(--height);
+  /* display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center; */
```

Comment:
You have to clean up your work from unnecessary lines of code (that's a good practice)